docs/getting-started/connect-widget.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: "Connect Widget"
+description: "Overview of the configurations for the widget."
+---
```

Comment:
Should we link to this from the Quickstart as well? ie "You can read more about the Connect Widget here" in a place that's appropriate